cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -0,0 +1,288 @@
+
+// TODO: Move the theme variables to cf-enhancements.
+/* topdoc
+  name: Theme variables
+  family: cf-core
+  notes:
+    - "The following color and sizing variables are exposed, allowing you to
+       easily override them before compiling."
+  patterns:
+    - name: Sizing
+      codenotes:
+        - |
+          @margin__em
+          @margin_half__em
+  tags:
+    - cf-core
+*/
+
+@margin__em: unit( @grid_gutter-width / @base-font-size-px, em );
+@margin_half__em: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
```

Comment:
That won't change anything. The output form unit will still be `1.875em`. If the font-size of the elem or a parent is set to something other than 16, the final calc in the browser will be incorrect. Just the world of EMs and why a lot of people prefer REMs.

If we want to be able to use these anywhere, we could convert them to `rem` instead of `em`, but we'd have to [do something else for IE](http://caniuse.com/#feat=rem). Possibly convert them to PX in the IE styles task?